cmake/projects/OpenSSL/schemes/url_sha1_openssl.cmake.in

```diff
@@ -33,6 +33,10 @@ else()
   set(configure_command "./config")
 endif()
 
+if(ANDROID)
+  set(configure_command MACHINE=${CMAKE_SYSTEM_PROCESSOR} RELEASE=2.6.37 SYSTEM=android ARCH=${ANDROID_ARCH_NAME} ${configure_command})
```

Comment:
> RELEASE is ignored but prevents it checking uname

Please add the comment about it.

> android toolchains set for ANDROID_ARCH_NAME. If not using a 'normal' android toolchain like polly, this may cause issues.

Since CMake 3.7 it will be [CMAKE_ANDROID_ARCH](https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARCH.html). We can be backward compatible here. First check CMAKE_ANDROID_ARCH, if defined - use it. Then ANDROID_ARCH_NAME. If none is defined emit error message by `hunter_user_error`.